apps/passport-server/src/services/telegramService.ts

```diff
@@ -279,17 +276,35 @@ export class TelegramService {
 
     this.bot.command("adminhelp", async (ctx) => {
       const messageThreadId = ctx?.message?.message_thread_id;
-      await ctx.reply(
-        `<b>Help</b>
-
-        <b>Admins</b>
-        <b>/manage</b> - Gate / Ungate this group with a ticketed event
-        <b>/setup</b> - When the chat is created, hide the general channel and set up Announcements.
-        <b>/incognito</b> - Mark a topic as anonymous
-      `,
-        { parse_mode: "HTML", reply_to_message_id: messageThreadId }
+      const admins = await ctx.getChatAdministrators();
```

Comment:
`/adminhelp` is now a command that admins can run in the #Admins topic and they will get a DM with the chat Id and the linked events in the database.